ocaml/testsuite/tests/typing-warnings/warning16.ml

```diff
@@ -6,7 +6,7 @@ let foo ?x = ()
 Line 1, characters 9-10:
 1 | let foo ?x = ()
              ^
-Warning 16 [unerasable-optional-argument]: this optional argument cannot be erased.
+Warning 16 [unerasable-omittable-argument]: this omittable argument cannot be erased.
```

Comment:
Is the rewording in this warning message more annoyance than it's worth?

Reply:
I don't much mind rewording the error output, though I am concerned about renaming the warning flag, as that's a user-visible incompatible change. Not really sure what's best here. It would be a good thing to raise on our team's Slack channel.